apps/constellation/CMakeLists.txt

```diff
@@ -6,14 +6,6 @@ project(fetch-constellation)
 
 setup_compiler()
 
-add_executable(constellation
```

Comment:
Adding the header files are useful for CLion

Reply:
ok

1.